google-login-plugin/src/com/google/gct/login/stats/UsageTrackerLoader.java

```diff
@@ -0,0 +1,45 @@
+package com.google.gct.login.stats;
+
+import com.intellij.notification.NotificationDisplayType;
+import com.intellij.notification.NotificationsConfiguration;
+import com.intellij.notification.NotificationsManager;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.components.ApplicationComponent;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Checks to see if user has opted in/out of usage tracking for the Cloud Tools plugin.
+ * If user has not, it notifies user to opt in/out of usage tracking, otherwise it does nothing.
+ */
+// TODO: Add tests
+public class UsageTrackerLoader implements ApplicationComponent {
+    @Override
+    public void initComponent() {
```

Comment:
Can this code go in the PluginInitializationComponent?  Potentially if we do that we could use the same logic path for conditionally adding usage tracking as we do for canCreateConfigurable in the GctConfigurationProvider.